src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionProvider.java

```diff
@@ -42,6 +42,7 @@
 import org.mule.runtime.extension.api.annotation.param.display.Summary;
 
 import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.lang.reflect.Proxy;
 import java.security.KeyManagementException;
```

Comment:
Can we remove the empty line 209? So the comment is just above the code. (I can't comment on that line, wasn't added by this commit) .